bootstrap-salt.sh

```diff
@@ -6619,8 +6620,8 @@ fi
 # Drop the master address if passed
 if [ "$_SALT_MASTER_ADDRESS" != "null" ]; then
     [ ! -d "$_SALT_ETC_DIR/minion.d" ] && mkdir -p "$_SALT_ETC_DIR/minion.d"
-    cat <<_eof > $_SALT_ETC_DIR/minion.d/99-master-address.conf
-master: $_SALT_MASTER_ADDRESS
+    cat <<_eof > "$_SALT_ETC_DIR/minion.d/99-master-address.conf
+master: $_SALT_MASTER_ADDRESS"
```

Comment:
I think here the quote must end on the file name, but "heredoc" content should remain unquoted.

Reply:
Oh, yep! Thanks!